src/lib/components/Tag.svelte

```diff
@@ -1,21 +1,56 @@
 <script lang="ts">
-  export let tagName: "span" | "h3" | "li" = "span";
+  export let tagName: "span" | "li" = "span";
+  export let intent: "warning" | "success" | "error" | "info" = "info";
+  export let size: "medium" | "large" = "medium";
+  export let testId = "tag";
 </script>
 
-<svelte:element this={tagName} data-tid="tag" class="tag"
+<svelte:element this={tagName} data-tid={testId} class={`tag ${size} ${intent}`}
   ><slot /></svelte:element
 >
 
 <style lang="scss">
   .tag {
-    padding: calc(var(--padding-0_5x) / 2) var(--padding);
+    border-radius: var(--border-radius-0_5x);
 
-    --tag-default-color: var(--value-color);
+    display: flex;
+    align-items: center;
+    gap: var(--padding-0_5x);
+    
+    // Force fit-content for `li` tags.
+    width: fit-content;
+    padding: var(--padding-0_5x) var(--padding);
 
-    color: var(--tag-color, var(--tag-default-color));
-    border: 1px solid var(--tag-color, var(--tag-default-color));
-    border-radius: var(--border-radius-5x);
+    &.large {
+      font-size: var(--font-size-standard);
```

Comment:
There isn't a mixin to use the font standard?

Reply:
I'm not sure I want to change the line height. Let me ask Artem.

Reply:
I think it makes sense to change the line-height. I'll use the mixin.